### PR TITLE
Towery - Mod2-Assignment1

### PR DESCRIPTION
**Pull request by jaamp**
Calculator

### CODE REVIEW

File: calculator.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+var results;
```
Comment (jimmynono) [inline calculator.js]: Don't use var, use const or let depending on your needs.

File: calculator.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+for (let i = 0; i < 3; i++) {
+
+    console.log('Round: ' + (i + 1));
+
+    var first = Math.ceil(Math.random() * 10);
+    var second = Math.ceil(Math.random() * 10);
+
+            results = first + second;
```
Comment (jimmynono) [inline calculator.js]: This whole block shouldn't be indented in this far, it should be aligned with lines 6 and 7

File: modualfourindex.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  <body>
+    <main>
+      <h1>JS Function</h1>
+      <h2>Mod 4</h2>
+      <h2>Assignment One</h2>
+    </main>
+    <script>
+      function assignmentOne() {
+      let num = prompt('Please enter a number.');
+      let sqNum = Math.pow(num, 2);
```
Comment (jimmynono) [inline modualfourindex.html]: Nice use of Math.pow!

File: excercise-one.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+const languages = ["Java", "C#", "Fortran", "HTML", "JavaScript", "COBOL"];
```
Comment (ClovisI) [inline excercise-one.js]: Need to add the following below this line:
console.log(languages);

File: excercise-one.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+languages.push("CSS");
```
Comment (ClovisI) [inline excercise-one.js]: Need to add the following below this line:
console.log(languages);

File: excercise-one.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+languages.splice(2, 1);
```
Comment (ClovisI) [inline excercise-one.js]: Need to add the following below this line:
console.log(languages);

**Pull request closed without merging**